cli/src/command.rs

```diff
@@ -133,6 +139,15 @@ pub enum InfoCommand {
     }
 }
 
+#[derive(Subcommand)]
+pub enum ManageSubcommand {
+    /// get a personalized webhook secret for a package
+    Webhook {
+        /// name of the package
+        name: String
```

Comment:
it would be cool if we'd have a --machine flag like on the serene secret subcommand